build.gradle

```diff
@@ -323,6 +323,7 @@ project('spring-integration-gemfire') {
 			exclude group: 'org.springframework', module: 'spring-core'
 			exclude group: 'org.springframework', module: 'spring-tx'
 		}
+		compile "commons-io:commons-io:$commonsIoVersion"
```

Comment:
Gemfire pulls in an older version (2.3) transitively which caused me problems in STS because the constructor we use for Tailer does not exist in 2.3. I am not sure why this suddenly happened; you added gemfire to -file last year. 

Gradle didn't let me add `exclude` for a project dep.

It seems to be an eclipse-only issue but, in any case, we don't want two version on the CP.

I did a full build with no issues (not testAll, though).